app/workers/users/estimate_default_language_worker.rb

```diff
@@ -1,6 +1,8 @@
 module Users
-  class EstimateDefaultLanguageJob < ApplicationJob
-    queue_as :users_estimate_language
+  class EstimateDefaultLanguageWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
 
     def perform(user_id, service = Users::EstimateDefaultLanguage)
```

Comment:
Since we are not using a different service here I think we can remove this argument and explicitly call this service below in the worker. 
```ruby
Users::EstimateDefaultLanguage.call(user) if user
```

Reply:
ok, I will change it